src/components/global/Footer.jsx

```diff
@@ -1,8 +1,208 @@
+        color: 'secondary.main'
+      }
+    }}
+  >
+    {icon}
+  </Link>
+));
+
+const socialBlobStyle = {
+  backgroundImage: 'url(/assets/socialsBlob.svg)',
+  backgroundSize: 'cover',
+  backgroundRepeat: 'no-repeat',
+  backgroundPosition: 'center',
+  overflow: 'visible',
+  minWidth: '42%',
+  minHeight: '175px'
+};
+
+const logoBlobStyle = {
+  backgroundImage: 'url(/assets/logoBlob.svg)',
+  backgroundRepeat: 'no-repeat',
+  backgroundPosition: 'center',
+  backgroundSize: 'cover',
+  overflow: 'visible',
+  minHeight: '40vh',
+  minWidth: '40vh'
+};
+
 const Footer = () => {
-  // any logic needed
-  // return (
-  // all the JSX (looks like HTML) to render
-  // )
+  const theme = useTheme();
+
+  // maps legalLinks used for placement in stack below
+  const renderLegalLinks = legalLinks.map((link, index) => (
+    <Box key={link.title + index}>
+      <Typography
+        key={link.title}
+        variant="body2"
+        color="tertiary.main"
+        sx={{
+          display: 'flex',
+          justifyContent: 'flex-end',
+          [theme.breakpoints.down('sm')]: {
+            p: '20px'
+          }
+        }}
+      >
+        {link.text ?? null}
+        <Link
+          href={link.href}
+          underline="none"
+          color="#000"
+          target={link.target ?? null}
+          rel={link.rel ?? null}
+          ml={link.ml ?? null}
+          sx={{
+            '&:hover': {
+              color: 'primary.main'
+            }
+          }}
+        >
+          {link.title}
+        </Link>
+      </Typography>
+    </Box>
+  ));
+
+  return (
+    // this box contains the entire footer
+    <Box
+      component="footer"
+      sx={{
+        display: 'inline-flex',
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        position: 'fixed',
+        bottom: 0,
+        flexDirection: 'row',
+        width: '100%',
+        background: 'lightgray',
+        [theme.breakpoints.down('sm')]: {
+          flexDirection: 'column'
+        }
+      }}
+    >
+      <Stack direction="row" alignItems="center">
+        <Box
+          sx={{
+            marginRight: 2,
+            background: logoBlobStyle,
+            [theme.breakpoints.down('sm')]: {
+              p: '40px'
+            }
+          }}
+          component="img"
+          alt="CODE PDX logo"
+          src={roseLogo}
+        />
+        <Box>
+          <Typography variant="h5">CODE PDX</Typography>
+        </Box>
+      </Stack>
+      {/* <Box sx={{ flexGrow: 1 }} /> */}
+      <Stack
+        direction="row"
+        spacing={3}
+        alignItems="center"
+        sx={{
+          ml: '50px',
+          mr: '50px',
+          socialBlobStyle,
+          [theme.breakpoints.down('sm')]: {
+            p: '40px'
+          }
+        }}
+      >
+        {renderSocialLinks}
+      </Stack>
+      {/* <Box sx={{ flexGrow: 2 }} /> */}
+      <Stack
+        sx={{
+          [theme.breakpoints.down('sm')]: {
```

Comment:
not sure about this

Reply:
Not sure what you are pointing at here but this code has also been changed already.

Reply:
True, but I don't think this stack needs padding on mobile at all - being that the `renderLegalLinks` have 20px padding already on mobile. 

If that changes, then possibly.